cabot/cabotapp/views.py

```diff
@@ -924,3 +929,182 @@ def get(self, *args, **kwargs):
             'pk': self.request.user.pk,
             'alerttype': kwargs['alerttype']
         }))
+
+
+class AckListView(LoginRequiredMixin, ListView):
+    model = Acknowledgement
+    context_object_name = 'acks'
+
+    def get_queryset(self):
+        return Acknowledgement.objects.filter(closed_at=None).order_by('status_check', '-id').prefetch_related()
```

Comment:
is `filter(x=None)` the canon way to filter for nulls?


use closed_at

Reply:
switched to `field__isnull=True`